otel-trace/config-user-service-otel-prod.properties

```diff
@@ -0,0 +1,4 @@
+otel.service.name=user-service
```

Comment:
`otel.service.name=user-service-otel` is better to distinguish OTEL SDK traces from Catchpoint Tracing SDK traces